src/include/network/terrier_server.h

```diff
@@ -83,6 +83,7 @@ class TerrierServer : public common::DedicatedThreadOwner {
   // For logging purposes
   // static void LogCallback(int severity, const char *msg);
 
+  const bool hot_standby_ = false;  // this server is currently in hot standby
```

Comment:
Why is this const

Reply:
This was added here to fix a compilation issue because the `ConnectionDispatcherTask` has been updated.
I am not sure but I think before the ITP protocol has been implemented, replication will be disabled so hot_standby_ would always be false. However, if needed I could also change the code to pass it as a parameter for starting a `TerrierServer`